src/re_graph/core.cljs

```diff
@@ -62,9 +62,9 @@
                                  (re-frame/dispatch (conj callback-event payload)))]}))))
 
 (defn query
-  ([query variables callback-fn] (query default-instance-name query variables callback-fn))
```

Comment:
Function and parameter have the same name, then function is shadowed in the scope